internal/util/bundleutil/bundleutil.go

```diff
@@ -0,0 +1,260 @@
+
+		if f.IsDir() {
+			// TODO(verify): we may have to log an error here instead of recursively copying
+			// if there are no subfolders allowed under manifests dir of a packagemanifest.
+			if err = copyOperatorManifests(srcPath, destPath); err != nil {
+				return err
+			}
+		} else {
+			srcFile, err := os.Open(srcPath)
+			if err != nil {
+				return err
+			}
+			defer srcFile.Close()
+
+			destFile, err := os.Create(destPath)
+			if err != nil {
+				return err
+			}
+			defer destFile.Close()
+
+			_, err = io.Copy(destFile, srcFile)
+			if err != nil {
+				return err
+			}
+		}
+	}
+	return nil
+}
+
+// BuildBundleImage builds the bundle image with the provided command or using
+// docker build command.
+func (meta *BundleMetaData) BuildBundleImage(tag string) error {
+
+	img := fmt.Sprintf("%s:%s", meta.BaseImage, tag)
+
+	if len(meta.BuildCommand) != 0 {
+		// TODO(varsha): Make this more user friendly by accepting a template which
+		// can executed in each bundle subdirectory.
+		log.Infof("Using the specified command to build image %s", img)
+		cmd := exec.Command(meta.BuildCommand, img)
+		if err := cmd.Run(); err != nil {
+			return err
+		}
+	} else {
+		output, err := exec.Command("docker", "build", "-f", filepath.Join(meta.BundleDir, "bundle.Dockerfile"), "-t", img, ".").CombinedOutput()
+		if err != nil || viper.GetBool(flags.VerboseOpt) {
+			fmt.Println(string(output))
+		}
+		if err != nil {
+			return err
+		}
+	}
+	log.Infof("Successfully built image %s", img)
+	return nil
+}
+
+// WriteScorecardConfig creates the scorecard directory in the bundle and copies the
+// configuration yaml to bundle.
+func (meta *BundleMetaData) WriteScorecardConfig(inputConfigPath string) error {
+	// If the config is already copied as a part of the manifest directory
+	// then ensure that it is deleted to remove duplicates.
+	_, filename := filepath.Split(inputConfigPath)
+	if err := deleteExistingScorecardConfig(meta.BundleDir, filename); err != nil {
+		return err
+	}
+
+	scorecardDir := filepath.Join(meta.BundleDir, "tests/scorecard")
+
+	// Create directory for scorecard config
+	if err := os.MkdirAll(scorecardDir, projutil.DirMode); err != nil {
+		return err
+	}
+
+	log.Info(fmt.Sprintf("Writing scorecard config in %s", scorecardDir))
+	b, err := ioutil.ReadFile(inputConfigPath)
+	if err != nil {
+		return err
+	}
+
+	err = ioutil.WriteFile(filepath.Join(scorecardDir, "config.yaml"), b, 0644)
+	if err != nil {
+		return fmt.Errorf("error writing scorecard config %v", err)
+	}
+	return nil
+}
+
+// deleteExistingScorecardConfig checks if there is an existing scorecard config file
+// in manifests/ folder, if present it deletes it.
+func deleteExistingScorecardConfig(bundleDir, filename string) error {
+	metadataDirPath := filepath.Join(bundleDir, defaultManifestDir, filename)
+	if _, err := os.Stat(metadataDirPath); !os.IsNotExist(err) {
+		if err := os.Remove(metadataDirPath); err != nil {
+			return err
+		}
+	}
+	return nil
```

Comment:
`os.RemoveAll()` will not return an error if the path does not exist.
```suggestion
	scorecardConfigPath := filepath.Join(bundleDir, defaultManifestDir, filename)
	return os.RemoveAll(scorecardConfigPath)
```